### PR TITLE
chore: BTC Staking testnet pages

### PR DESCRIPTION
**Pull request by gusin13**
- Makes a new dir 
- Puts finality and btc-staker pages inside this dir
- Changes bbn-test-2 to bbn-test-3
- Changes mainnet to signet
- Remove notes about test keying restriction
- Removes notes about bls sigs to be submitted as txs at epoch end or details about tx fee due to bls sigs
- Renames finality prov to become a final prov

### CODE REVIEW

File: docs/user-guides/btc-staking-testnet/btc-staker.md [hunk 1]
```diff
@@ -0,0 +1,459 @@
+---
+id: btc-staker
+title: BTC Staker
+sidebar_label: BTC Staker
+---
+
+BTC-Staker is a toolset designed for seamless Bitcoin staking.
```
Comment (vitsalis) [inline docs/user-guides/btc-staking-testnet/btc-staker.md:7]: In the BTC staker docs, this sentence is under `Overview`. If you make any updates, please make sure to make them on the Github repo first and then copy paste here.

File: docs/user-guides/btc-staking-testnet/finality-providers/overview.md [hunk 2]
```diff
@@ -1,13 +1,9 @@
...
+title: Become a Finality Provider
 sidebar_label: Finality Providers
 ---
 
-:::caution
-This software is unreleased; CLI options may change in the official release.
-:::
-
 A toolset crafted for the creation and
 management of Finality Providers.
```
Comment (vitsalis) [inline docs/user-guides/btc-staking-testnet/finality-providers/overview.md:9]: Shouldn't we point the users to where the software is located or provide them instructions for cloning etc.? These should probably be added to the github repo and copy pasted here.

Same for BTC staker.

File: docs/user-guides/btc-timestamping-testnet/setup-node.md [hunk 3]
```diff
@@ -64,7 +64,6 @@ go install cosmossdk.io/tools/cosmovisor/cmd/cosmovisor@latest
 
 Create the necessary directories
 ```console
-mkdir -p ~/.babylond
```
Comment (vitsalis) [inline docs/user-guides/btc-timestamping-testnet/setup-node.md:67]: Why?

File: sidebars.js [hunk 4]
```diff
@@ -57,16 +57,30 @@ const sidebars = {
...
           items: [
-            'user-guides/finality-providers/eots-manager',
-            'user-guides/finality-providers/finality-provider',
-          ],
-        }
+            'user-guides/btc-staking-testnet/setup-node',
+            'user-guides/btc-staking-testnet/getting-funds',
+            'user-guides/btc-staking-testnet/become-validator',
+            'user-guides/btc-staking-testnet/btc-staker',
+            {
```
Comment (vitsalis) [inline sidebars.js:70]: Finality Provider should be before BTC staker..

File: docs/user-guides/btc-timestamping-testnet/setup-node.md (hunk 3 above)
Comment (filippos47) [reply docs/user-guides/btc-timestamping-testnet/setup-node.md:67]: It's already created when you init the babylon node

File: sidebars.js (hunk 4 above)
Comment (gusin13) [reply sidebars.js:69]: fixed

**Review by vitsalis — APPROVED** [review]

**Pull request merged**